CHANGE-NOTES.md

```diff
@@ -6,6 +6,7 @@
 - Improved Override Fork Point dialog: inapplicable options are hidden, placeholders are no longer displayed.
 - Fixed an error with a too long GitHub URI when reporting exceptions in the plugin.
 - Fixed a too short traverse info dialog.
+- Clarified Slide In dialog: a branch to slide in can be created on the spot
```

Comment:
I would try to stick to the words Added/Fixed/Improved (Improved in this case ofc) - no strong feelings